dawn/src/dawn/SIR/SIR.h

```diff
@@ -559,7 +559,7 @@ class Global : public Value, NonCopyable {
   Global(Kind type) : Value(type) {}
 };
 
-using GlobalVariableMap = std::unordered_map<std::string, Global>;
+using GlobalVariableMap = std::map<std::string, Global>;
```

Comment:
Do you think it's appropriate to add a comment here that this should really be an unordered_map but we're changing it to ordered to fix the codegen tests? If so, please add!

Reply:
Sure, I can add a comment.

Reply:
Added!